src/semantik.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+lol
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Refactor or remove this statement. ([javascript:S905](https://sonarcloud.io/organizations/migueldelmazo-github/rules#rule_key=javascript%3AS905))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Semantik&pullRequest=1&issues=AWQJgFlN23Tx-xDbkDx_&open=AWQJgFlN23Tx-xDbkDx_)

[ISSUE_KEY]: AWQJgFlN23Tx-xDbkDx_